src/projects/navigation/project-pull-request-list-item.html

```diff
@@ -81,16 +102,25 @@
       </div>
       <div class="main">
         <list-item-label type="title">
-        <a href$="{{_cistatus.target_url}}" target="_blank"><iron-icon title="CI passed" class$="icon-passed [[_cistatus.state]]" icon="icons:check-circle"></iron-icon></a>
-        <a href$="{{_cistatus.target_url}}" target="_blank"><iron-icon title="CI failed" class$="icon-failed [[_cistatus.state]]" icon="icons:cancel"></iron-icon></a>
+          <span class="ci-placeholder" hidden$="[[_hasCIStatus]]"></span>
+          <span hidden$="[[!_hasCIStatus]]">
+            <a href$="{{_cistatus.target_url}}" target="_blank">
+              <iron-icon title="CI passed" class$="icon-passed [[_cistatus.state]]" icon="icons:check-circle"></iron-icon>
+            </a>
+            <a href$="{{_cistatus.target_url}}" target="_blank">
+              <iron-icon title="CI failed" class$="icon-failed [[_cistatus.state]]" icon="icons:cancel"></iron-icon>
+            </a>
+          </span>
```

Comment:
This is quite a bulk of html for just a placeholder. Would it not be better to keep the previous HTML and add an attribute to the two `iron-icon`s with some css to toggle their opacity?

It would look something like this:
```
<iron-icon has-status="[[_hasCIStatus]]" ... />

iron-icon {
  opacity: 0;
}
iron-icon[has-status] {
  opacity: 1;
}
```